tests/keras/test_sequential_model.py

```diff
@@ -129,6 +129,12 @@ def data_generator(x, y, batch_size=50):
     model.fit(x_train, y_train, batch_size=batch_size, epochs=epochs, verbose=2, validation_split=0.1)
     model.fit(x_train, y_train, batch_size=batch_size, epochs=epochs, verbose=0)
     model.fit(x_train, y_train, batch_size=batch_size, epochs=epochs, verbose=1, shuffle=False)
+    try:
+        import pandas as pd
+
+        model.fit(pd.DataFrame(x_train), pd.DataFrame(y_train), batch_size=batch_size, epochs=epochs, verbose=0)
```

Comment:
If you are adding pandas support to more methods than `fit`, then it should be tested too.

Reply:
Also: line too long